cristin-import/src/main/java/no/unit/nva/cristin/mapper/artisticproduction/CristinArtisticProduction.java

```diff
@@ -9,15 +9,41 @@
 import lombok.Builder;
 import lombok.Getter;
 import lombok.Setter;
+import no.unit.nva.cristin.mapper.nva.exceptions.InvalidIsrcException;
 import no.unit.nva.model.contexttypes.UnconfirmedPublisher;
 import no.unit.nva.model.contexttypes.place.UnconfirmedPlace;
 import no.unit.nva.model.instancetypes.artistic.film.MovingPicture;
 import no.unit.nva.model.instancetypes.artistic.film.realization.MovingPictureOutput;
 import no.unit.nva.model.instancetypes.artistic.film.realization.OtherRelease;
+import no.unit.nva.model.instancetypes.artistic.music.AudioVisualPublication;
+import no.unit.nva.model.instancetypes.artistic.music.Concert;
+import no.unit.nva.model.instancetypes.artistic.music.InvalidIsmnException;
+import no.unit.nva.model.instancetypes.artistic.music.Ismn;
+import no.unit.nva.model.instancetypes.artistic.music.Isrc;
+import no.unit.nva.model.instancetypes.artistic.music.MusicMediaSubtype;
+import no.unit.nva.model.instancetypes.artistic.music.MusicMediaType;
+import no.unit.nva.model.instancetypes.artistic.music.MusicPerformance;
+import no.unit.nva.model.instancetypes.artistic.music.MusicPerformanceManifestation;
+import no.unit.nva.model.instancetypes.artistic.music.MusicScore;
+import no.unit.nva.model.instancetypes.artistic.music.MusicalWork;
+import no.unit.nva.model.instancetypes.artistic.music.MusicalWorkPerformance;
+import no.unit.nva.model.instancetypes.artistic.music.OtherPerformance;
+import no.unit.nva.model.time.Instant;
 import nva.commons.core.JacocoGenerated;
+import nva.commons.core.StringUtils;
 
+import java.util.ArrayList;
+import java.util.Map;
+import java.util.Optional;
 
-/**
+import static java.util.Objects.nonNull;
+import static no.unit.nva.model.instancetypes.artistic.music.MusicMediaType.COMPACT_DISC;
+import static no.unit.nva.model.instancetypes.artistic.music.MusicMediaType.DIGITAL_FILE;
+import static no.unit.nva.model.instancetypes.artistic.music.MusicMediaType.OTHER;
+import static no.unit.nva.model.instancetypes.artistic.music.MusicMediaType.VINYL;
+import static nva.commons.core.attempt.Try.attempt;
+
+/*
```

Comment:
Is it a javadoc or comment?

Reply:
it just a comment for me (or others) to keep track of what secondary category has this object attached to it.